### ISSUE TITLE
Add support for setting a password

### ISSUE THREAD

**Issue by rmelotte** [QuantumEntangledAndy/neolink#303]
Currently it is possible to enable the HTTP/s port using neolink, which makes it possible to configure and use the camera with external platforms (HomeAssistant, Frigate, ...) without having to ever use the Reolink proprietary application.

Setting the password however is currently missing from neolink (or is it just me not able to find it?) and cannot initially be done from the web UI (on a brand new Reolink E1 Zoom at least, there is no password by default and the "change password" feature requires one to be set first - password "Can't be blank" when changing it).

Implementing a password set function through the same mechanism the official application uses would make it possible to use the cameras without ever depending on the proprietary application.

**Comment by QuantumEntangledAndy** [comment]
The default password in most cameras is `123456` have you tried that for the webui. But yes we don't have the change password protocol REd (never had a need to)

**Comment by rmelotte** [comment]
Sorry for not replying.

The problem, at least with newer cameras, is that the default password is blank.
When going to the management web interface to set a password, it complains that the old password cannot be blank:

![Image](https://github.com/user-attachments/assets/488e3029-33a2-40c9-8802-5ad680518b89)

So as a result, when buying a new camera users are left with an insecure camera with no password set and no possibility to set it without using the official application.

However in the meantime I tried with the new version that includes https://github.com/QuantumEntangledAndy/neolink/pull/345/, and it works!

I could set a password this way on a CX810:
```
neolink  --config=/etc/neolink.toml  users <Camera> password admin '<PASSWORD>'
```

So in other words: we can close this issue.

Thanks a lot @ecksun and @QuantumEntangledAndy !
